php/wp-settings-cli.php

```diff
@@ -166,6 +192,8 @@
 
 // Load the L10n library.
 require_once ABSPATH . WPINC . '/l10n.php';
+maybe_require( '4.6-alpha-38496', ABSPATH . WPINC . '/class-wp-locale.php' );
```

Comment:
```suggestion
Utils\maybe_require( '4.6-alpha-38496', ABSPATH . WPINC . '/class-wp-locale.php' );
```

@schlessera shouldn't this also be using `Utils`?

Reply:
Yes, indeed, just fixed that via https://github.com/wp-cli/wp-cli/pull/5300/commits/b9f53a71e894bf5203d224f0d4ea8de74b2ab93c